src/main/java/org/openml/webapplication/generatefolds/GenerateFolds.java

```diff
@@ -19,172 +19,164 @@
-					splits.add(am.createInstance(true,rowid,r,f));
+					splits.add(am.createInstance(true, rowid, r, f));
 				}
-				for( int i = 0; i < test.numInstances(); ++i ) {
+				for (int i = 0; i < test.numInstances(); ++i) {
 					int rowid = (int) test.instance(i).value(0);
-					splits.add(am.createInstance(false,rowid,r,f));
+					splits.add(am.createInstance(false, rowid, r, f));
 				}
 			}
 		}
 		return splits;
 	}
-	
-	private Instances sample_splits_leaveoneout( String name ) {
-		Instances splits = new Instances(name,am.getArffHeader(),splits_size);
-		for( int f = 0; f < dataset.numInstances(); ++f ) {
-			for( int i = 0; i < dataset.numInstances(); ++i ) {
+
+	private Instances sample_splits_leaveoneout(String name) {
+		Instances splits = new Instances(name, am.getArffHeader(), splits_size);
+		for (int f = 0; f < dataset.numInstances(); ++f) {
+			for (int i = 0; i < dataset.numInstances(); ++i) {
 				int rowid = (int) dataset.instance(i).value(0);
-				splits.add(am.createInstance(f!=i,rowid,0,f));
+				splits.add(am.createInstance(f != i, rowid, 0, f));
 			}
 		}
 		return splits;
 	}
-	
-	private Instances sample_splits_learningcurve( String name ) {
-		Instances splits = new Instances(name,am.getArffHeader(),splits_size);
-		for( int r = 0; r < evaluationMethod.getRepeats(); ++r ) {
+
+	private Instances sample_splits_learningcurve(String name) {
+		Instances splits = new Instances(name, am.getArffHeader(), splits_size);
+		for (int r = 0; r < evaluationMethod.getRepeats(); ++r) {
 			dataset.randomize(rand);
-			if (dataset.classAttribute().isNominal())
+			if (dataset.classAttribute().isNominal()) {
 				InstancesHelper.stratify(dataset); // do our own stratification
+			}
 			
-			for( int f = 0; f < evaluationMethod.getFolds(); ++f ) {
+			for (int f = 0; f < evaluationMethod.getFolds(); ++f) {
 				Instances train = dataset.trainCV(evaluationMethod.getFolds(), f);
 				Instances test = dataset.testCV(evaluationMethod.getFolds(), f);
-				
-				for( int s = 0; s < EstimationProcedure.getNumberOfSamples( train.numInstances() ); ++s ) {
-					for( int i = 0; i < EstimationProcedure.sampleSize( s, train.numInstances() ); ++i ) {
+
+				for (int s = 0; s < getNumberOfSamples(train.numInstances()); ++s) {
+					for (int i = 0; i < sampleSize(s, train.numInstances()); ++i) {
 						int rowid = (int) train.instance(i).value(0);
-						splits.add(am.createInstance(true,rowid,r,f,s));
+						splits.add(am.createInstance(true, rowid, r, f, s));
 					}
-					for( int i = 0; i < test.numInstances(); ++i ) {
+					for (int i = 0; i < test.numInstances(); ++i) {
 						int rowid = (int) test.instance(i).value(0);
-						splits.add(am.createInstance(false,rowid,r,f,s));
+						splits.add(am.createInstance(false, rowid, r, f, s));
 					}
 				}
 			}
 		}
 		return splits;
 	}
 	
-	private Instances sample_splits_bootstrap( String name ) throws Exception {
+	/* private Instances sample_splits_bootstrap( String name ) throws Exception {
```

Comment:
Remove?

Reply:
No, I think we need these in the future. For now, these are commented out since they are not properly tested.